src/azure-cli/azure/cli/__main__.py

```diff
@@ -22,6 +23,12 @@
 
 logger = get_logger(__name__)
 
+AZURELINUX2_WARNING_MESSAGE = (
+    "Azure CLI 2.74.0 is the last version available on Azure Linux (Mariner) 2.0 and will not receive updates. "
+    "Consider migrating to Azure Linux 3.0 to use newer versions of Azure CLI. "
+    "To disable this warning message, set AZURE_CLI_DISABLE_AZURELINUX2_WARNING environment variable to any value."
```

Comment:
https://github.com/Azure/azure-cli/pull/29628 has a sentence:

> For more information: https://go.microsoft.com/fwlink/?linkid=2282203

But I can't find any public document for Azure Linux 2.0's EOL. https://github.com/microsoft/azurelinux doesn't mention this. There is only an internal document: https://eng.ms/docs/products/azure-linux/overview/20eol. https://github.com/microsoft/azurelinux/issues/13555 was raised for this issue.